### ISSUE TITLE
Docs: explain group vs. layer

### ISSUE THREAD

**Issue by pgilfernandez** [friction2d/friction#334]
It might be not a bug as groups and layers are not the same but as groups have the `opacity` parameter exposed it would have to work...

![group-layer](https://github.com/user-attachments/assets/89af8e05-3f23-4dbc-959d-24e4c3e3cd96)

example project: [opacity.friction.zip](https://github.com/user-attachments/files/17815566/opacity.friction.zip)

So, the `opacity` parameter doesn't work for groups, it's not something important but trying to follow a good UX, it would suggest to:
- hide the `opacity` parameter in Groups
- make it works

Thanks!

**Comment by rodlie** [comment]
Groups are not layers.

Opacity is part of transform, it does not know what the parent box/item/whatever is.

**Comment by pgilfernandez** [comment]
> Groups are not layers.
> 
> Opacity is part of transform, it does not know what the parent box/item/whatever is.

I understand what you mean, but if, under a group, that property doesn't make sense (or does nothing), it should be hidden (for instance) or it could confuse users, don't you think?

BTW, this is not an important thing, but when you decide to fine polish Friction, this could be addressed...

**Comment by rodlie** [comment]
Yeah... but this is also a slippery slope, the more complexity added the worse things get over time.

If the code change is trivial sure.

**Comment by rodlie** [comment]
btw, it does serve a purpose as SVG supports opacity in groups.

**Comment by pgilfernandez** [comment]
> Yeah... but this is also a slippery slope, the more complexity added the worse things get over time.
> 
> If the code change is trivial sure.

Yes, I agree with that, hehehe

> btw, it does serve a purpose as SVG supports opacity in groups.

Once again you are right, but then, why to to add opacity support for groups inside Friction?

**Comment by pgilfernandez** [comment]
I understand... well, if you fully feel like this is not an issue then close it  ;)

**Comment by pgilfernandez** [comment]
> The best option IMHO is documentation, a dedicated "Groups vs. Layers" section probably.

Hahaha, once again you are right, this is very important. I hope to help in this regard =)

**Comment by rodlie** [comment]
Can we close this? Or do we need to add something to the doc? (don't remember 😄 )

**Comment by pgilfernandez** [comment]
We can close it. I already wrote about it ☺️
